### PR TITLE
Delete parameter store lookup

### PR DESCRIPTION
**Pull request by nicl**
Parameter Store now supports easy (substring) search so we don't
need our own version. 🎈 

Behaviour no longer required is:

Screenshot 2021-08-05 at 09 55 49

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@
 ## [Features and components](06-features-and-components/)
 - [SteadyPage JS utility](06-features-and-components/01-steadypage-js-utility.md)
 - [How are trail pictures picked in Frontend?](06-features-and-components/02-trail-images.md)
+- [theguardian.com/font-loader](06-features-and-components/03-font-loader-route.md)
```
Comment (jamesgorrie) [inline docs/README.md:61]: Is this meant to be part of this PR?

**Review by jamesgorrie — APPROVED** [review]
We can remove this too:
https://github.com/guardian/frontend/blob/main/dev-build/conf/routes#L194-L195

**Review by OllysCoding — APPROVED** [review]
+1

File: docs/README.md (hunk 1 above)
Comment (nicl) [reply docs/README.md:61]: Hmm, not sure why it is here. I can see it on `main` though so not too concerned. I'll rebase in any case which might disappear it.

**Pull request closed without merging**